main.tf

```diff
@@ -0,0 +1,81 @@
+# Password auto generate
+resource "random_password" "password" {
+  count   = var.password == "" ? 1 : 0
+  length  = 16
+  special = true
+}
+
+module "postgresql" {
+  source = "github.com/opszero/terraform-aws-rds"
+
+  #lables
+  name        = var.name
+  environment = var.environment
+  managedby   = var.managedby
+  repository  = var.repository
+  label_order = var.label_order
+
+  #networking
+  allowed_ip          = var.allowed_ip
+  allowed_ports       = var.allowed_ports
+  vpc_id              = var.vpc_id
+  subnet_ids          = var.subnet_ids
+  publicly_accessible = var.publicly_accessible
+
+  #storage
+  allocated_storage = var.allocated_storage
+  storage_encrypted = true
+
+  #backups
+  backup_retention_period = 7
+  backup_window           = "03:00-06:00"
+  maintenance_window      = "Mon:00:00-Mon:03:00"
+
+  #logs
+  enabled_cloudwatch_logs_exports = var.enabled_cloudwatch_logs_exports
+
+
+
+  #database
+  db_name        = var.db_name
+  username       = var.username
+  password       = var.password == "" ? join("", random_password.password.*.result) : var.password
```

Comment:
I'd like to use this instead of this code. Can you change the rds code: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#managed-master-passwords-via-secrets-manager-default-kms-key